bottom/unpack.py

```diff
@@ -224,29 +224,32 @@ def unpack_command(msg: str) -> Tuple[str, Dict[str, Any]]:
         nickmask(prefix, kwargs)
         kwargs["new_nick"] = params[0]
 
-    elif command == 'RPL_NAMREPLY':
+    elif command in ["RPL_NAMREPLY"]:
         kwargs["target"] = params[0]
-        kwargs["channel_type"] = params[1] if len(params) > 3 else None
+        if len(params) > 3:
+            kwargs["channel_type"] = params[1]
+        else:
+            kwargs["channel_type"] = None
```

Comment:
This looks like another case where we don't exactly follow RFC 2812. The channel type is required in 2812, but isn't a thing in 1459, so it has to be optional in order to support both. I don't think an empty string is correct here since this argument *isn't* actually optional as an empty string might imply. I think the better option would be to just not set it at all, but None is a good alternative that doesn't break anything.